internal/controller/pdbwatcher_controller.go

```diff
@@ -3,311 +3,117 @@ package controllers
 import (
```

Comment:
Can we leave this file along and just add this two controllers in this pr? if theres a good reason for doing this same time let me know here and I'll try and review this refactor a bit more closely.

Reply:
I am assuming this is a new controller and the original controller is moved to podautoscaler. if you give the new controller a new file and leave existing one as is it will be much easier to review